convert2rhel/checks.py

```diff
@@ -532,13 +579,18 @@ def is_loaded_kernel_latest():
         if match == 0:
             logger.info("Kernel currently loaded is at the latest version.")
         else:
+            repos_message = (
+                "on the enabled system repositories"
+                if not reposdir
+                else "on repositories defined in the %s folder" % reposdir
```

Comment:
```suggestion
                "in the enabled system repositories"
                if not reposdir
                else "in repositories defined in the %s folder" % reposdir
```
Sorry if I made it confusing in the comment https://github.com/oamg/convert2rhel/pull/458#discussion_r883345313 but the `in` preposition was correct here (there isn't `based` before it here).
tests/integration/tier1/system-up-to-date/test_non_latest_kernel_inhibitor.py